### DISCUSSION TITLE
How can i get the current userid in Session

### DISCUSSION THREAD

**Discussion by sojinsamuel** [teamhanko/hanko#1107] [Q&A] [answered]
I want the user ID of the current logged-in user, so that i could have the same ids in my database too as of hanko. But i couldn't find a method that does this in the docs. Can the Hanko community help, please?

Likewise, the email of the logged-in user too, if that's possible. I am using the free Hanko account. I am preparing for the Hanko hackathon.

**Comment by FlxMgdnz** [comment, marked as answer]
Hey, check out this guide: https://docs.hanko.io/guides/users/get-user-data

**Reply by sojinsamuel** [reply]
Thanku very much @FlxMgdnz
